### ISSUE TITLE
Keeps ignoring changes directly to HTML file

### ISSUE THREAD

**Issue by karlclement** [sitecake/sitecake#67]
I added a `<footer>` tag manually to my HTML file and saved it. Then i made a few content changes to the page and clicked Publish. The footer was removed from my page. Is there a way of clearing the 'cache' to show my latest changes?

Thanks,

**Comment by nikolamircic** [comment]
Yes sure. Once you start editing you website Sitecake will create `sitecake-backup` folder. Go in this folder and look for a subfolder with date and time you want to roll back. Just copy/paste files you need.

**Comment by karlclement** [comment]
@nikolamircic I don't to rollback, I am want it to stop removing my footer and recent changes directly to HTML when I publish changes with sitecake. I tried deleting the sitecake-backup folder but it didn't fix it.

**Comment by sitecake** [comment]
Every time Sitecake starts an editing session it checks for `sitecake-temp` folder (in the site root folder). If  `sitecake-temp` folder is there, Sitecake uses HTML located there. Basically, that folder contains the work from the previous editing session.
But, if there's no `sitecake-temp` folder, Sitecake creates it using the content in the root folder (HTML files).
Every time the Publish butting is hit, Sitecake replaces the content in the root folder (that's why your `<footer>` tag goes away).
To preserve your changes (made directly in HTML in the root folder), simply remove `sitecake-temp` folder before starting a new editing session. And be careful not to remove it if there is some unpublished content.
